lib/cli.js

```diff
@@ -275,6 +275,7 @@ const meowOptions = {
 		ignorePattern: {
 			alias: 'ip',
 			type: 'string',
+			isMultiple: true,
```

Comment:
[suggestion] How about improving the type-checking as follows?
When I try running the `npm run lint:types` command, it passes.

```diff
diff --git a/lib/cli.js b/lib/cli.js
index c4562fd0..05465559 100644
--- a/lib/cli.js
+++ b/lib/cli.js
@@ -32,7 +32,7 @@ const EXIT_CODE_ERROR = 2;
  * @property {string} [help]
  * @property {boolean} [ignoreDisables]
  * @property {string} [ignorePath]
- * @property {string} [ignorePattern]
+ * @property {string[]} [ignorePattern]
  * @property {string} [noColor]
  * @property {string} [outputFile]
  * @property {boolean} [stdin]
@@ -76,7 +76,7 @@ const EXIT_CODE_ERROR = 2;
  * @property {boolean} [disableDefaultIgnores]
  * @property {number} [maxWarnings]
  * @property {string} [syntax]
- * @property {any} [ignorePattern]
+ * @property {string[]} [ignorePattern]
  * @property {boolean} [allowEmptyInput]
  * @property {string} [files]
  * @property {string} [code]
diff --git a/lib/standalone.js b/lib/standalone.js
index 98c18b8c..8903f7ce 100644
--- a/lib/standalone.js
+++ b/lib/standalone.js
@@ -68,10 +68,6 @@ module.exports = function (options) {
 		if (readError.code !== FILE_NOT_FOUND_ERROR_CODE) throw readError;
 	}
 
-	/**
-	 * TODO TYPES
-	 * @type {any}
-	 */
 	const ignorePattern = options.ignorePattern || [];
 	const ignorer = ignore().add(ignoreText).add(ignorePattern);
 
diff --git a/lib/utils/getFileIgnorer.js b/lib/utils/getFileIgnorer.js
index 2b20a498..ac07789c 100644
--- a/lib/utils/getFileIgnorer.js
+++ b/lib/utils/getFileIgnorer.js
@@ -27,10 +27,6 @@ module.exports = function (options) {
 		if (readError.code !== FILE_NOT_FOUND_ERROR_CODE) throw readError;
 	}
 
-	/**
-	 * TODO TYPES
-	 * @type {any}
-	 */
 	const ignorePattern = options.ignorePattern || [];
 	const ignorer = ignore().add(ignoreText).add(ignorePattern);
 
diff --git a/types/stylelint/index.d.ts b/types/stylelint/index.d.ts
index db105352..bbdede5a 100644
--- a/types/stylelint/index.d.ts
+++ b/types/stylelint/index.d.ts
@@ -145,7 +145,7 @@ declare module 'stylelint' {
 		printConfig?: string;
 		ignoreDisables?: boolean;
 		ignorePath?: string;
-		ignorePattern?: RegExp;
+		ignorePattern?: string[];
 		reportNeedlessDisables?: boolean;
 		reportInvalidScopeDisables?: boolean;
 		maxWarnings?: number;
```